book/src/dev/rfcs/0012-value-pools.md

```diff
@@ -24,25 +24,25 @@ Checking the coins created by coinbase transactions and funding streams is out o
 # Definitions
 [definitions]: #definitions
 
-- `value balance` - The total change in value caused by a subset of the blockchain.
-- `transparent value balance` - The change in the value of the transparent pool. The sum of the outputs spent by transparent inputs in `tx_in` fields, minus the sum of newly created outputs in `tx_out` fields.
-- `coinbase transparent value balance` - The change in the value of the transparent pool due to a coinbase transaction. The coins newly created by the block, minus the sum of newly created outputs in `tx_out` fields. In this design, we temporarily assume that all coinbase outputs are valid, to avoid checking the created coins.
-- `sprout value balance` - The change in the sprout value pool. The sum of all sprout `vpub_old` fields, minus the sum of all `vpub_new` fields.
+- `value balance` - The change in the chain value pools, caused by a subset of the blockchain.
+- `transparent value balance` - The change the transparent value pool. The sum of newly created outputs in `tx_out` fields, minus the sum of the outputs spent by transparent inputs in `tx_in` fields.
+- `coinbase transparent value balance` - The change the transparent value pool, due to a coinbase transaction. The sum of newly created outputs in `tx_out` fields.
```

Comment:
This change swaps the transparent sign in the transparent value balance.

It also sets coinbase inputs to zero, which we already did in the code.